setup.py

```diff
@@ -23,7 +23,7 @@
         ]
     },
     include_packages=True,
-    python_requires=">=3.7.7, <3.11",
+    python_requires=">=3.7.7, <=3.11",
```

Comment:
@jdebacker. Parallel to the change in `environment.yml` you should probably remove the `<=3.11` here in `setup.py`.